components/function-controller/pkg/apis/serverless/v1alpha2/function_types.go

```diff
@@ -249,13 +249,13 @@ type Repository struct {
 
 // FunctionStatus defines the observed state of Function
 type FunctionStatus struct {
-	Runtime              Runtime     `json:"runtime,omitempty"`
-	Conditions           []Condition `json:"conditions,omitempty"`
-	Repository           `json:",inline,omitempty"`
-	Replicas             int32  `json:"replicas,omitempty"`
-	PodSelector          string `json:"podSelector,omitempty"`
-	Commit               string `json:"commit,omitempty"`
-	RuntimeImageOverride string `json:"runtimeImageOverride,omitempty"`
```

Comment:
This field should be marked as `deprecated` and we can remove it in `v1alpha3`.
We shouldn't introduce any breaking changes.

Reply:
Added as you suggested